docs/tutorials/spotify.md

```diff
@@ -14,23 +16,29 @@ The Spotify tutorial will show you how to use the powerful HTTPClient class to p
 * None
 
 #### Steps
-++1++ Create a new empty Creo project and start by dragging a TableView from Objects to Window1. In TableView1 Cell Inspector section set Height to 90 and Template to Subtitle (in this example we will not create a CustomView object but we'll use a built-in one, a Subtitle template gives an Image, a Title and a Subtitle field):
+++1++ Create a new empty Creo project and start by dragging a TableView from Objects to Window1. 
+Set the TableView's Properties as:
+1. Height:  '90'
+2. Template: 'Subtitle'.
+
+You can also create your own custom view template. But for this example we will not do that. The Subtitle template gives an Image, a Title and a Subtitle field.
```

Comment:
The Subtitle template provides (instead of gives) an Image, a Title and a Subtitle field.